src/components/analytics/ExampleAnalyticsIntegration.tsx

```
@@ -0,0 +1,260 @@
+          onClick={handleApprove}
+          isLoading={isApproving}
+          loadingText="Approving..."
+          onMouseEnter={() => handleTooltipOpen('approve-button-tooltip')}
+          onClick={() => handleButtonClick('approve-button', 'primary')}
+        >
+          Approve {tokenSymbol}
+        </Button>
+        
+        <Button
+          colorScheme="green"
+          onClick={handleDeposit}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)